.github/workflows/pytest.yml

```diff
@@ -64,9 +63,11 @@ jobs:
             --ignore=pymc/tests/test_distributions.py
             --ignore=pymc/tests/test_distributions_random.py
             --ignore=pymc/tests/test_idata_conversion.py
+            --ignore=pymc/tests/test_smc.py
+            --ignore=pymc/tests/test_bart.py
+            --ignore=pymc/tests/test_missing.py
 
           - |
-            pymc/tests/test_initial_point.py
```

Comment:
This test module is to short to justify placing it manually (everything takes ~10 seconds locally)